study.md

```diff
@@ -160,15 +166,27 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+Responses is the result of making a request to the http server.  It
+contains protocol/version, status code, description, headers, and body.  Some common response codes are:
+404:  file is no longer available
+403: forbidden.  Your machine does not have permissions to view (hasn't been whitelisted)
+500:  Internal server error.  It's a server-side catchall error.  You might get a different result if you try again.
+503:  Service unavailable.  Web server is unavailable.  It is likely temporary.
+
+https://www.globo.tech/learning-center/5-most-common-http-error-codes-explained/
+http://geekexplains.blogspot.com/2008/06/whats-http-explain-http-request-and.html
+https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
+https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+in the terminal, enter $ curl http://www.reddit.com
+http://www.thegeekstuff.com/2012/04/curl-examples/?utm_source=feedburner
```

Comment:
close we have to specify the type of request too

`curl --request GET https://www.reddit.com`